src/lint/linter/UberCheckstyleLinter.php

```diff
@@ -134,7 +137,7 @@ private function getCommand() {
     $this->checkConfiguration();
     $futures = array();
     // Call checkstyle in batches of 500
-    $chunks = array_chunk($paths, 500);
+    $chunks = array_chunk($paths, 100);
```

Comment:
Why did the batch size change here?

Can we just make this configurable, as this will make changes to the android monorepo much slower